src/test/files/java/ConfigFriendlyNameCheckMultipleConfig.java

```diff
@@ -0,0 +1,25 @@
+import org.mule.api.annotations.components.MetaDataCategory;
+
+import org.mule.api.annotations.Connector;
+
+@ConnectionManagement(friendlyName = "Configuration")
+public class MyConnectorBasicAuthConfig {
+
+}
+
+@OAuth(friendlyName = "OAuth 1.0")
+public class MyConnectorOAuth1Config {
+
+}
+
+@OAuth2(friendlyName = "OAuth v2.0 Configuration")
+public class MyConnectorOAuth2Config { // Noncompliant {{For multiple connector configurations, if one or more are annotated with @OAuth2, the default friendly name must be 'OAuth
+                                       // 2.0' (instead of 'OAuth v2.0 Configuration').}}
```

Comment:
I don't think this can be broken down into 2 lines... I believe tests are not working